graph-manager-docs/source/integrations.md

```diff
@@ -142,22 +142,29 @@ If you're reporting metrics to Graph Manager through the Engine proxy, Datadog w
 
 #### Setup
 
-Getting set up with Graph Manager's Datadog integration is as simple as providing a Datadog API key to Graph Manager. There's no further configuration required! You will need access to a Datadog account with administrator privileges to acquire an API key.
+Getting set up with the Apollo Datadog integration is as simple as providing a Datadog API key to Graph Manager. There's no further configuration required! You will need access to a Datadog account with administrator privileges to acquire an API key.
 
-1.  Go to the [Datadog integrations page](https://app.datadoghq.com/account/settings) and search for "Apollo Engine".
-2.  Click the "+Available" button and go the the _Configuration_ tab. Copy the API key from the "Configuration" tab, click "Install Integration" at the bottom, and go to the [service](https://engine.apollographql.com) you'd like to enable Datadog Metric Forwarding for.
-3.  In the settings for the service, scroll to "Integrations", and toggle Datadog to ON. When prompted, paste in the API key.
-4.  Go to your Datadog metric explorer and start to see the metrics flow in! Please allow up to five minutes for metrics to be visible.
+1. Go to the [Datadog integrations page](https://app.datadoghq.com/account/settings) and search for "Apollo".
 
-Navigate to the Apollo Engine Integration in Datadog
+![DatadogIntegrationTile](./img/datadog/integration-tile.png)
 
-![IntegrationTile](./img/datadog/integration-tile.png)
+2. Press the **+Available** button on the Apollo tile, and go the _Configuration_ tab. Scroll to the bottom of the tab, and press the **Install Integration** button.
```

Comment:
I'm not sure our users really need this level of detail, unless we think the datadog integrations UI is so bad it merits detailed explanation

Reply:
The UI is pretty bad. Specifically, if you hover over the Apollo tile in Datadog, the **+Available** button changes to a **+Install** button. But clicking that button doesn't actually install the integration.

To actually install the integration, you have to click the Apollo tile, go to the Configuration tab, scroll down and click **+Install Integration**. If you want it to be less words, I could just say

> 2. Click the Apollo tile, go to the _Configuration_ tab, scroll down and press **+Install Integration**.

Reply:
Updated it a bit more, see https://github.com/apollographql/apollo/pull/733#discussion_r359497862 .